pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -176,10 +178,15 @@ let
     LIBRARY_PATH = makeLibraryPath paths;
   };
 
+  # Python 2.7 needs this
+  crossCompileEnv = if stdenv.hostPlatform != stdenv.buildPlatform
```

Comment:
You can simplify this with optionalAttrs.

Reply:
Thanks. Will do!